core/src/com/unciv/ui/pickerscreens/ModManagementScreen.kt

```diff
@@ -417,6 +427,29 @@ class ModManagementScreen(
         val label = if (installedModInfo[repo.name]?.state?.hasUpdate == true)
             "Update [${repo.name}]"
         else "Download [${repo.name}]"
+
+        if (!repo.hasUpdatedSize) {
+            Concurrency.run("GitHubParser") {
+                try {
+                    val repoSize = Github.getRepoSize(repo)
+                    Log.debug("Total size: $repoSize")
+
+                    if (repoSize > 0f) {
+                        launchOnGLThread {
+                            repo.size = repoSize.toInt()
+                            repo.hasUpdatedSize = true
+                            if (selectedMod == repo)
+                                updateModInfo()
+                        }
+                    }
+
+                } catch (e: IOException) {
+                    /* ... */
```

Comment:
?

Reply:
If parsing fails for whatever reason, we just catch exception and do nothing